src/Driver/Driver.jl

```diff
@@ -210,16 +210,21 @@ function setup_solver(t0::FT, timeend::FT,
                       init_args...;
                       forcecpu=false,
                       ode_solver_type=nothing,
+                      ode_dt=nothing,
+                      modeldata=nothing,
                       Courant_number=0.4,
                       T=FT(290)
                      ) where {FT<:AbstractFloat}
     @tic setup_solver
 
     # create DG model, initialize ODE state
-    dg = DGModel(driver_config.bl, driver_config.grid, driver_config.numfluxnondiff,
-                 driver_config.numfluxdiff, driver_config.gradnumflux)
+    dg = DGModel(driver_config.bl, driver_config.grid,
+                 driver_config.numfluxnondiff, driver_config.numfluxdiff,
+                 driver_config.gradnumflux, modeldata=modeldata)
     @info @sprintf("Initializing %s", driver_config.name)
     Q = init_ode_state(dg, FT(0), init_args...; forcecpu=forcecpu)
+    update_aux!(dg, driver_config.bl, Q, FT(0))
+    update_aux_diffusive!(dg, driver_config.bl, Q, FT(0))
```

Comment:
These seem out of place. Does `init_aux()` not suffice?

Reply:
`update_aux!` applies the filters to the state variables and `update_aux_diffusive!` calculates the vertical component of velocity using the integral kernels. 

Both of these features probably deserve their own interface instead of being done during the `update_aux` calls, but I think adding those interfaces is a few steps down the road